src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -543,6 +543,14 @@
 
     public static final Key<Value<Vector3d>> LEFT_ARM_ROTATION = KeyFactory.fake("LEFT_ARM_ROTATION");
 
+    /**
+     * Represents the {@link Key} for representing the dominant {@link HandType}
+     * of a {@link Living} entity.
```

Comment:
@gabizou `EntityLivingBase` (`Living`) contains the `getPrimaryHand` method - all living entities can be handed, it's just up to the implementations to handle the **setting** of the hand.